README.md

```diff
@@ -105,12 +108,23 @@ When all services have fully started, `nvidia-smi` should show processes like th
 |    3   N/A  N/A   1355929      C   ...s/python/triton_python_backend_stub      424MiB |
 |    3   N/A  N/A   1373202      C   tritonserver                                414MiB |
 +---------------------------------------------------------------------------------------+
-
 ```
 If it's taking > 1m for `nvidia-smi` to return, it's likely the bus is still busy setting up the models.
 
 Once it completes normally (less than a few seconds), the NIM models are ready.
 
+You can observe the started containers with `docker ps`:
```

Comment:
Shouldn't the NIM containers be shown in this output?

Reply:
good catch. Missed those in the output.